build.gradle

```diff
@@ -1,7 +1,8 @@
-apply plugin: 'application'
+plugins {
```

Comment:
This PR is now conflicting with master. Also check [this comment](https://github.com/openjfx/openjfx-docs/pull/7#discussion_r223447269) about using buildscript instead of plugin block.